base/docs/helpdb/Base.jl

```diff
@@ -923,13 +923,23 @@ Stacktrace:
 BoundsError
 
 """
-    invoke(f, types <: Tuple, args...)
+    invoke(function, ArgumentTypes::Type, arguments...)
 
 Invoke a method for the given generic function matching the specified types, on
 the specified arguments. The arguments must be compatible with the specified types. This
 allows invoking a method other than the most specific matching method, which is useful when
 the behavior of a more general definition is explicitly needed (often as part of the
 implementation of a more specific method of the same function).
+
+# Examples
+```jldoctest
+julia> f(x) = x;
+
+julia> f(x::Integer) = 1 + invoke(f, Tuple{Any}, x);
```

Comment:
To be clear that the `f` above accepts arguments of type `Any`, perhaps be explicit and change the above method to
```julia
julia> f(x::Any) = x
```
?

Or perhaps even better, update to something like
```julia
julia> f(x::Real) = x;

julia> f(x::Integer) = 1 + invoke(f, Tuple{Real}, x);

julia> f(2)
3
```